lib/node_modules/@stdlib/lapack/blas/dlaswp/lib/dlaswp.js

```diff
@@ -0,0 +1,133 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isLayout = require( '@stdlib/blas/base/assert/is-layout' );
+
+
+// MAIN //
+
+/**
+* Performs a series of row interchanges on the matrix `A`.
+*
+* @param {PositiveInteger} N - number of columns of `A`
+* @param {Array} A - matrix
+* @param {PositiveInteger} LDA - leading dimension of `A`
+* @param {PositiveInteger} k1 - first element of `IPIV` for which a row interchange will be done
+* @param {PositiveInteger} k2 - second element of `IPIV` for which a row interchange will be done
```

Comment:
This description seems off, or at least it doesn't make sense to me. Not clear how `(k2-k1+1)` implies that `k2` is the second element for row interchange.

Reply:
I tried to update it, let me know this makes sense now.

Reply:
Yes, I think it makes more sense. Slightly awkward that this is the way BLAS specifies the range of values in IPIV to iterate over, as a user could provide a `k2` which isn't a multiple of `k1*incrX`.